StackExchange.Redis/StackExchange/Redis/GeoEntry.cs

```diff
@@ -0,0 +1,144 @@
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace StackExchange.Redis
+{
+
+    /// <summary>
+    /// Describes a GeoEntry element with the corresponding value
+    /// GeoEntries are stored in redis as SortedSetEntries
+    /// </summary>
+    public struct GeoEntry : IEquatable<GeoEntry>, IEquatable<SortedSetEntry>, IComparable, IComparable<GeoEntry>, IComparable<SortedSetEntry>
```

Comment:
(writing as I work down the commit, so initial thoughts) - this looks a lot like `SortedSetEntry`; but I can get behind having a separate struct if it has a valid different semantic meaning; does it actually need to be equatable / comparable to `SortedSetEntry` though? is there an API overlap that this is catering for?